docs/tools/checkout/configuration.md

```diff
@@ -89,10 +90,16 @@ The `callToAction` object lets you customize the messages displayed on the check
 
 (Make sure you use a number and not a string!)
 
-- `1`: mainnet,
-- `4`: rinkeby,
-- `100`: xdai,
-- `137`: polygon.
+- `1`: mainnet
+- `4`: rinkeby
+- `5`: goerli
+- `10`: optimism
+- `56`: binance
+- `100`: xdai
+- `137`: polygon
+- `42220`: celo
+
+Always check our [networks](../../core-protocol/unlock/networks) page for the most up to date list of supported networks.
```

Comment:
I think we should remove the list above as it will be hard to maintain and just point (like you did) to the right page

Reply:
Removed.